src/components/study-container.js

```diff
@@ -542,44 +556,57 @@ export function StudyContainer({ view, onChangeTab }) {
     }, [intl]);
 
     useEffect(() => {
-        if (studyUpdatedForce.eventData.headers) {
-            if (
-                studyUpdatedForce.eventData.headers[UPDATE_TYPE_HEADER] ===
-                'study'
-            ) {
-                // study partial update :
-                // loading equipments involved in the study modification and updating the network
-                const substationsIds =
-                    studyUpdatedForce.eventData.headers['substationsIds'];
-                const tmp = substationsIds.substring(
-                    1,
-                    substationsIds.length - 1
-                ); // removing square brackets
-                if (tmp && tmp.length > 0) {
-                    updateNetwork(tmp.split(', '));
-                }
+        if (!mapManualRefresh || !reloadMapNeeded) {
```

Comment:
I am not sure if the modifications made in this file are relevant. If the network is not reloaded when currentNode changes or when network has been updated on the back side, the data displayed in the equipment table might be wrong.

Reply:
I implemented a fix which reload network data when the user switch to the spreadsheet
If the user is in manual mode, the spreadsheet is already open, and new data comes in therefore the data is invalidated